sale_order_no_create_partner/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 PESOL>
+# # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "Sale Order no create partner",
+    'summary': "Avoid creating a new partner in the sales order section",
+    'version': '12.0.1.0.0',
+    'category': 'Sales',
+    'website': 'https://github.com/OCA/',
```

Comment:
You should point the repo. 
```suggestion
    'website': 'https://github.com/OCA/sale-workflow',
```